### DISCUSSION TITLE
✅ Add Option for Custom Icons in Elementor Search Form

### DISCUSSION THREAD

**Discussion by daniyal-ostechhub** [elementor/elementor#24532] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The current version of Elementor lacks the ability to customize icons in the search form, limiting the visual diversity and personalization options for users. While the default icons are functional, users often seek a more tailored experience that aligns with their website's unique design.

### Describe the solution you'd like

I propose the addition of an option in Elementor's search form settings to allow users to upload or select custom icons for their search bars. This enhancement would empower users to align their search forms with the overall aesthetic of their websites, providing a more cohesive and personalized user experience.

### Describe alternatives you've considered

CSS Customization: While users can currently customize the search form through custom CSS, having a built-in option for custom icons would simplify the process and make it more accessible to users with varying levels of technical expertise.

Third-Party Plugins: Some users may resort to third-party plugins to achieve custom icons in the search form. However, integrating this feature directly into Elementor would eliminate the need for additional plugins, streamlining the user experience and reducing potential conflicts.

### Additional context

In modern web design, customization plays a pivotal role in creating unique and engaging user interfaces. Allowing users to easily incorporate custom icons into the Elementor search form would enhance the platform's flexibility and cater to the diverse needs of its user base.

Thank you for considering this feature request. I believe that the addition of custom icons to the Elementor search form would be a valuable enhancement, providing users with more creative control over the appearance of their websites.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/), and select your own icons, it also includes the option to choose your query as well!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
